example_test.go

```diff
@@ -1,27 +1,30 @@
 package secureheader_test
 
 import (
-	"github.com/kr/secureheader"
 	"net/http"
+
+	"github.com/kr/secureheader"
 )
 
 func Example() {
 	http.Handle("/", http.FileServer(http.Dir("/tmp")))
-	http.ListenAndServe(":80", secureheader.DefaultConfig)
+	http.ListenAndServe(":80", secureheader.Handler(nil))
 }
 
 func Example_custom() {
 	http.Handle("/", http.FileServer(http.Dir("/tmp")))
-	secureheader.DefaultConfig.HSTSIncludeSubdomains = false
-	secureheader.DefaultConfig.FrameOptions = false
-	http.ListenAndServe(":80", secureheader.DefaultConfig)
+	h := secureheader.Handler(http.DefaultServeMux)
```

Comment:
Articles like [this](https://blog.gopheracademy.com/advent-2016/exposing-go-on-the-internet/) mention `http.DefaultServeMux` as a potential security gotcha. Would it be better to use something else here in the context of a security-focused library?

Reply:
Yeah maybe. For the purpose of this PR, I just wanted to update the form of the examples, but not change their behavior. Opened #15 to track this.